contracts/LiquidityTree.sol

```diff
@@ -138,7 +138,9 @@ contract LiquidityTree {
         push(updatedNode, begin, end, leaf, ++updateId);
 
         // remove amount (percent of amount) from leaf to it's parents
-        withdrawAmount = (treeNode[leaf].amount * percent) / DECIMALS;
+        withdrawAmount = uint128(
```

Comment:
Hi! Thank you for you review, this issue fixing by previous PR #3

Reply:
Ahh cool, i see :smile: